spec/services/load_statements_spec.rb

```diff
@@ -258,5 +258,30 @@
         expect(statement.electoral_district_item).to eq 'Q789'
       end
     end
+
+    context 'use an EveryPolitician CSV as the upstream source' do
+      let(:suggestions_store_response) do
+        <<~CSV
+          id,name,sort_name,email,twitter,facebook,group,group_id,area_id,area,chamber,term,start_date,end_date,image,gender,wikidata,wikidata_group,wikidata_area
+          c216f406-b11f-4f2b-a890-e9922293478e,Anthony Martinez,Anthony Martinez,,,,United Democratic Party,90ee4fde-68d6-4544-bb2a-2a58371b629f,7764e1c2-da2a-456f-9ab1-d66665df244d,Port Loyola,House of Representatives,9,,,http://nationalassembly.gov.bz/images/house_of_rep/a_martinez.jpg,male,Q4773030,Q1809323,Q16975862
```

Comment:
I haven't really looked closely at the implementation here, but might it be worth also adding a test row where at least one of the wikidata columns is blank? A majority of CSVs don't have all the fields pre-reconciled every time, and I can imagine scenarios where that might trip something up (even if this code is perfectly fine with it, we'd want to be defensive against something breaking with later refactoring)

Reply:
Yes good idea. Done in 18347e6